jest.config.js

```diff
@@ -0,0 +1,11 @@
+module.exports = {
+    setupFilesAfterEnv: ['jest-enzyme', '<rootDir>/src/test/setupTests.js'],
+    testEnvironment: 'enzyme',
+    verbose: true,
+    testURL: 'http://localhost/',
+    testRegex: '(/__tests__/.*|(\\.|/)(test|spec))\\.(jsx?|tsx?)$',
```

Comment:
Simplify a bit the tests matching, and remove the limit for `__tests__` directory:
```suggestion
    testMatch: ['<rootDir>/**/?(*.)+(spec|test).(js|ts)?(x)'],
```